spotbugs/build.properties

```diff
@@ -1,9 +1,13 @@
 source.spotbugs.jar = src/main/java/,\
-                      src/gui/,\
+                      src/gui/main/,\
+                      src/gui/test/java/,\
                       src/test/java/,\
                       src/xsl/,\
                       etc
 bin.includes = .libs/,\
                META-INF/
 jars.compile.order = spotbugs.jar
+output.spotbugs.jar = bin/main/,\
+					  bin/gui/,\
```

Comment:
Please use spaces rather than tabs, for consistency with other parts of the file.